### ISSUE TITLE
Liqui fetch_ticker

### ISSUE THREAD

**Issue by marinsokol5** [ccxt-dev/ccxt#234]
Using latest ccxt version in python.

When i try to fetch ticker in async branch from Liqui
```
liqui = ccxt.liqui()
loop = asyncio.get_event_loop()
loop.run_until_complete(liqui.fetch_ticker('ETH/BTC' ))
```

I get:
```
  File "/usr/lib/python3.6/asyncio/base_events.py", line 467, in run_until_complete
    return future.result()
  File "/home/marin/fino/.venv/lib/python3.6/site-packages/ccxt/async/exchanges.py", line 12198, in fetch_ticker
    tickers = await self.fetchTickers([id])
TypeError: fetchTickers() takes 1 positional argument but 2 were given
```

Method fetch tickers in Liqui :
```
 async def fetch_ticker(self, symbol):
        await self.load_markets()
        market = self.market(symbol)
        id = market['id']
        tickers = await self.fetchTickers([id])
        return tickers[symbol]
```

Should be changed to 
```
 async def fetch_ticker(self, symbol):
        await self.load_markets()
        market = self.market(symbol)
        id = market['id']
        tickers = await self.fetchTickers()
        return tickers[id]
```
or...

Cheers friend.

**Comment by kroitor** [comment]
Hello, @marinsokol5 ! As always, good to hear from you! Thanks a lot for your feedback, it's very helpful to us! I'll take a look and fix the issue with Liqui fetchTickers asap.

**Comment by kroitor** [comment]
Hmm... That is strange... I tried the following, and it worked:

```shell
mbp:ccxt igorkroitor$ python3
Python 3.6.2 (v3.6.2:5fd33b5926, Jul 16 2017, 20:11:06) 
[GCC 4.2.1 (Apple Inc. build 5666) (dot 3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import ccxt.async as ccxt
>>> import asyncio
>>> print(asyncio.get_event_loop().run_until_complete(ccxt.liqui().fetch_ticker('ETH/BTC')))
{'timestamp': 1506368046000, 'datetime': '2017-09-25T19:34:06.000Z', 'high': 0.08141084, 
'low': 0.07421783, 'bid': 0.07546336, 'ask': 0.0760007, 'vwap': None, 'open': None, 'close': None, 
'first': None, 'last': 0.07558197, 'change': None, 'percentage': None, 'average': 0.077814335, 
'baseVolume': 6392.17138498, 'quoteVolume': 493.25031144543055, 'info': {'high': 0.08141084, 
'low': 0.07421783, 'avg': 0.077814335, 'vol': 493.25031144543055, 'vol_cur': 6392.17138498, 
'last': 0.07558197, 'buy': 0.07546336, 'sell': 0.0760007, 'updated': 1506368046}}
>>> ccxt.__version__
'1.8.14'
>>>
```

I also added an example here: https://github.com/ccxt-dev/ccxt/blob/master/examples/py/async-fetch-ticker.py

```Python
# -*- coding: utf-8 -*-

import asyncio
import os
import sys
import ccxt.async as ccxt  # noqa: E402

print(asyncio.get_event_loop().run_until_complete(ccxt.liqui().fetch_ticker('ETH/BTC')))
```

Does it still fail to work for you with ccxt 1.8.14+ ?

**Comment by kroitor** [comment]
I found a transpiler bug there... I noticed in your output that it was calling the non-redefined version of fetchTicker, whereas it should have called fetch_ticker as per python PEP8 notation. And I fixed it, the new version 1.8.20+ should work fine. We will be thankful if you report back on this. Thx again!
